pom.xml

```diff
@@ -65,7 +65,7 @@
     <dependency>
       <groupId>org.mirah</groupId>
       <artifactId>mirah-complete</artifactId>
-      <version>0.1.1</version>
+      <version>0.1.3</version>
```

Comment:
It's now up!

On Mon, Mar 23, 2015 at 7:42 AM, Nick Howard <ndh@baroquebobcat.com> wrote:

> Hmm. I thought I'd published it. I must have missed a step.
>
> On Mon, Mar 23, 2015 at 3:02 AM, uujava <notifications@github.com> wrote:
>
>> In pom.xml
>> <https://github.com/mirah/maven-mirah-plugin/pull/8#discussion_r26920938>
>> :
>>
>> > @@ -65,7 +65,7 @@
>> >      <dependency>
>> >        <groupId>org.mirah</groupId>
>> >        <artifactId>mirah-complete</artifactId>
>> > -      <version>0.1.1</version>
>> > +      <version>0.1.3</version>
>>
>> Mirah 0.1.4 version could be set in plugin configuration. I'm using
>> following plugin config with 0.1.4 published in my local artifactory:
>>
>> <plugin>
>>                     <groupId>org.mirah.maven</groupId>
>>                     <artifactId>maven-mirah-plugin</artifactId>
>>                     <version>1.2-SNAPSHOT</version>
>>                     <executions>
>>                         <execution>
>>                             <id>mirah-compile</id>
>>                             <phase>compile</phase>
>>                             <goals>
>>                                 <goal>compile</goal>
>>                             </goals>
>>                         </execution>
>>                         <execution>
>>                             <id>mirah-test-compile</id>
>>                             <phase>test-compile</phase>
>>                             <goals>
>>                                 <goal>testCompile</goal>
>>                             </goals>
>>                         </execution>
>>                     </executions>
>>                     <configuration>
>>                         <verbose>false</verbose>
>>                         <newClosures>false</newClosures>
>>                     </configuration>
>>                     <dependencies>
>>                       <dependency>
>>                         <groupId>org.mirah</groupId>
>>                         <artifactId>mirah-complete</artifactId>
>>                         <version>0.1.4</version>
>>                       </dependency>
>>                     </dependencies>
>>                 </plugin>
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mirah/maven-mirah-plugin/pull/8/files#r26920938>.
>>
>
>
>
> --
> -Nick Howard
> http://blog.baroquebobcat.com/
>
>


-- 
-Nick Howard
http://blog.baroquebobcat.com/